docs/graphql/core/scheduled-triggers/create-one-off-scheduled-event.rst

```diff
@@ -131,3 +131,11 @@ Once you've created your scheduled trigger, you can see ``Pending events``, ``Pr
 .. thumbnail:: /img/graphql/core/event-triggers/pending-one-off.png
    :alt: Schedule and logs for scheduled events
    :width: 1200px
+
+
+One minute precision
+--------------------
+
+This means that for any scheduled event set at a particular time, it actually delivers on the 0th second of the next minute.
```

Comment:
This is not accurate and doesn't fit with the example below. It's a bit confusing, because the issue description is wrong. Karthikeyan added a comment to explain: https://github.com/hasura/graphql-engine/issues/5754. So it will be delivered *within one minute* of when it's scheduled, *not* at the 0th second of the next minute.